jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/BrowserOpen.java

```diff
@@ -0,0 +1,26 @@
+package io.github.epam.html.tests.elements.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.githubLink;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertEquals;
+
+public class BrowserOpen implements TestsInit {
+
+    String text = "Github JDI";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4665121)